frontend/app/components/Logo.vue

```diff
@@ -0,0 +1,7 @@
+<template>
+  <NuxtImg
```

Comment:
Either include svg here (for example nuxt ui website does that), or remove this component because now we don't use diffferent colors.